### DISCUSSION TITLE
Dumb question : how to have my lights ON on start

### DISCUSSION THREAD

**Discussion by Leeoneil-Vely** [missionpinball/missionpinball#28] [Q&A]
Hello everyone !
i'm sorry but i feel dumb, but i'm trying to get all my lights on at the beginning of a mode (base mode) and I don't understand how to do it simply.

I'm currently using the tutorial, i've done a nice attract lights show (so all my leds are ok and configured) with the monitor and the showcreator.
But I don't understand where to put the information (and how) to lights all the led (or just one led) when i start a game.

Can you direct me to the right section or help me ?
I've done some research here (on the mpf group), I probably don't understand something very simple !
I'm stuck here, sorry if it supposed to be easy...

Thank's for your help !

**Comment by Leeoneil-Vely** [comment]
I'm sorry, i found something : _light_player:_ 
I don't know if it's the way to do it but it's working.
I've tagged all my LED with "gi" and activated them with a color on my base mode.

Sorry to have created a subject for this.
You can delete it if you want !

**Comment by colemanomartin** [comment]
Glad you found it.  Best to leave the post here so others with the same puzzle can benefit from your knowledge.

**Comment by Leeoneil-Vely** [comment]
In fact, I don't know if it's the good way to do it.
Because it cause all the LED flashing between 2 balls (all the led are off a brief instant when the ball is drain).

I placed my _light_player_ in the base mode config.yaml, it look like this.

light_player:
  mode_base_started:
    gi:
      color: white

Maybe I need to add a light transition or something like that when the ball is drained ?
